### DISCUSSION TITLE
Documentation

### DISCUSSION THREAD

**Discussion by rodolfomiranda** [zakaio/atala-prism-schema#7] [Q&A] [answered]
Do you have more documentation that the ppt shared in TH? It's a good idea to put it in the repo

**Comment by rssh** [comment]
In process.  Currently, we have 
- definitions themselves:  https://github.com/zakaio/atala-prism-schema/blob/main/json_schema/meta/prism-credentials.json
- I thinking about finding a literate programming tool,  which will generate good docs with commented sources. Or will write by hands.
- also in tools/ts-schema tools incomplete generator of json_schema and json-ld context

**Comment by rssh** [comment, marked as answer]
Btw, I  have setup gh-pages:  https://zakaio.github.io/atala-prism-schema/
 Now we have a description, which is automatically generated for schema; other parts will follow with time ;)

**Reply by rodolfomiranda** [reply]
nice!
